$resaction need in 5.0.x too

defrance opened the pull request.

defrance committed:
$resaction need in 5.0.x too

hregis commented:
@defrance it's wrong !
if you use $resaction here, it can crush the $resaction of hook type "addreplace" !!

defrance commented:
@hregis you have already this correction on development branch
you don't have see the else just before the addreplace
without this change, you don't have the $result on hook returned and testable

hregis commented:
@defrance ok ok !!! :-)
c'est la faute de la chaleur !!

eldy commented on htdocs/core/class/hookmanager.class.php:
```diff
@@ -233,7 +233,7 @@ function executeHooks($method, $parameters=false, &$object='', &$action='')
 
                     	//dol_syslog("Call method ".$method." of class ".get_class($actionclassinstance).", module=".$module.", hooktype=".$hooktype, LOG_DEBUG);
-                    	$result = $actionclassinstance->$method($parameters, $object, $action, $this); // $object and $action can be changed by method ($object->id during creation for example or $action to go back to other action for example)
+                    	$resaction = $actionclassinstance->$method($parameters, $object, $action, $this); // $object and $action can be changed by method ($object->id during creation for example or $action to go back to other action for example)
```
If you change result into resaction action, you must also change result in following lines

defrance committed:
Changes Done

question
need to change $actionclassinstance->results in $actionclassinstance->resactionss ???
